src/layouts/PageLayout/PageHeader/PageHeader.module.scss

```diff
@@ -0,0 +1,26 @@
+/* Copyright (c) Fortanix, Inc.
+|* This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of
+|* the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+@use '../../../styling/defs.scss' as bk;
+
+@layer baklava.components {
+  .bk-page-header {
+    @include bk.component-base(bk-page-header);
+    
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    
+    .bk-page-header__actions {
+      display: flex;
+      flex-direction: row;
+      gap: bk.$spacing-7;
+    }
+    
+    .bk-page-header__select > div {
+      border-block-end-width: 0;
+    }
```

Comment:
Removing the line here breaks focus/active states (e.g. you can't see focus because that's done by making the line blue).

But also, I think these specs might be wrong/outdated. Let's keep the `Select` with its basic styling for now.